src/lib/env.ts

```diff
@@ -26,10 +26,16 @@ try {
 } catch {
   stringifiedFallbackOvalConfigs = undefined;
 }
+// Chain ID and network
+const chainId = getInt(getEnvVar("CHAIN_ID", fallback.chainId.toString()));
+if (chainId !== 1 && chainId !== 5) throw new Error(`Unsupported chainId: ${chainId}`);
```

Comment:
Better check against mapping in constants if we support the chain. If testing on Sepolia this would start getting too long here